realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/transport/OsJavaNetworkTransportTests.kt

```diff
@@ -105,7 +106,11 @@ class OsJavaNetworkTransportTests {
                 } else {
                     fail("Unexpected request url: $url")
                 }
-                return Response.httpResponse(200, successHeaders, result)
+                return OkHttpNetworkTransport.OkHttpResponse.httpResponse(200, successHeaders, result)
+            }
+
+            override fun sendStreamingRequest(request: Request): Response {
+                TODO("Not yet implemented")
```

Comment:
Throw an exception here instead. Leaving a TODO is confusing. Same in the other cases below.